Library/Formula/libmemcached.rb

```diff
@@ -15,7 +13,11 @@ class Libmemcached < Formula
   end
 
   option "with-sasl", "Build with sasl support"
+  option "with-static", "Build static library"
+  option "with-memaslap", "Build with load generation and benchmark tool support"
+  option "with-hsieh-hash", "Build with hsieh hashing support"
 
+  depends_on "libevent" # requires --enable-libmemcachedprotocol
```

Comment:
@DomT4 Judging by the output of ./configure, it requests it anyway libevent. Besides RHEL distributions use the default libmemcachedprotocol for full coverage, such as php*-memcached includes such possible additional options.